docs/architecture/lb/nginx.md

```diff
@@ -0,0 +1,270 @@
+# NGINX
+
+NGINX is a popular solution, but there are important disadvantages to consider when using it for scaling syslog ingestion:
+
+- **Uneven TCP traffic distribution**: Even with round-robin load balancing, TCP traffic may not be evenly distributed, leading to overloaded instances. This can cause growing queues, delays, data loss, and potential memory or disk issues.
+  
+- **UDP limitations**: As UDP is a lossy protocol, it’s recommended to send data directly from the source to a nearby syslog server. Using a load balancer can introduce another point of data loss.
+  
+- **Lack of active health checking**: NGINX Open Source does not provide active health checking, which is important for UDP Direct Server Return (DSR) load balancing. NGINX Plus offers active health checking but requires a paid license.
+  
+- **No built-in High Availability (HA)**: NGINX Open Source lacks native support for High Availability. Without HA, your NGINX load balancer becomes a single point of failure. NGINX Plus includes built-in HA support, but it is part of the paid offering.
+
+**Please note that Splunk only supports SC4S**. If issues arise due to the load balancer, please reach out to the NGINX support team.
+
+## Install NGINX Open Source
+
+Refer to the NGINX documentation for instructions on installing NGINX **with the stream module**, which is required for TCP/UDP load balancing. For example, on Ubuntu:
```

Comment:
Refer to the NGINX documentation to install NGINX **with the stream module**, which is required for TCP/UDP load balancing. For example, on Ubuntu: